Symmetry-Adapted-Perturbation-Theory/SAPT0_no_S2.py

```diff
@@ -0,0 +1,264 @@
+# A script to compute the SAPT0 interaction energy without the the Single-Exchange Approximation
+# 
+# Created by: Jonathan M. Waldrop
+# Date: 1/9/2018
+# License: GPL v3.0
```

Comment:
Can you update to a header similar to: https://github.com/psi4/psi4numpy/blob/master/Self-Consistent-Field/RHF_DIIS.py#L5?